### PR TITLE
#124 add Monoid formula

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #124

### CODE REVIEW

**Review by ronaldtse — CHANGES_REQUESTED** [review]
Can we add the homepage to the formula? Thanks!

**Comment by CAMOBAP** [comment]
> Can we add the homepage to the formula? Thanks!

Done

File: Formulas/monoid.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - family_name: Monoid
+    type: Retina
+    full_name: Monoid Retina
+    post_script_name: Monoid-Retina
+    version: '0.62'
+    copyright: Copyright (c) 2015, @larsenwork
+    font: Monoid-Retina.ttf
+extract:
+  format: zip
+copyright: Copyright (c) 2016, Andreas Larsen
```
Comment (ronaldtse) [inline Formulas/monoid.yml]: Let's change this to what the README provides:
> Copyright (c) 2015, Andreas Larsen and contributors.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
@CAMOBAP Given that many open source fonts have open source repositories, maybe we should add a metadata attribute that tells people where the font source is at? Thoughts?

**Comment by CAMOBAP** [comment]
> @CAMOBAP Given that many open source fonts have open source repositories, maybe we should add a metadata attribute that tells people where the font source is at? Thoughts?

Good idea, I think we can add `source_repository`/`sources`/`repository` after `homepage` property

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thanks @CAMOBAP ! Feel free to merge anytime.

**Pull request merged**